cypress/integration/organisation/create.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Creating an organisation
+  Background:
+    Given the user is logged in
+  
+  Scenario: The user should be able to click the button to create a new organisation
+    Given the user is on the organisation page
+     When the user clicks on the create new organisation button
+     Then the user should be on the create new organisation page
+  
+  @organisation
+  Scenario: The user should be able to create a new organisation
+    Given the user is on the create new organisation page
+     When the user enters the organisation information
```

Comment:
i would prefer Given, When, Then to have the same starting line

Reply:
@k-joseph thanks for the review. Let me fix right away.

Reply:
@kaweesi I actually prefer this style. It keeps the actual steps aligned with each other, which I think is more useful for figuring out what's important. As a bonus, it also keeps `And` indented under the `Given`, `When`, `Then` which makes it easier, at least to me, to follow whats going on. In short, for me at least, it's easier to visually see what each scenario does.